### PR TITLE
Sql function transformation

### PR DESCRIPTION
**Pull request by MariaDolotova**

### CODE REVIEW

File: UI/src/app/components/pages/mapping/mapping.component.html [hunk 1]
```diff
@@ -123,6 +123,18 @@
               <circle cx="10" cy="10" r="8" fill="#066BBB" />
               <text x="7" y="14" style="fill:rgb(255, 255, 255); font-size: 11;">L</text>
             </marker>
+            <marker
```
Comment (costebano) [inline UI/src/app/components/pages/mapping/mapping.component.html:126]: marker-end-M also should be added

File: UI/src/app/components/sql-transformation/sql-transformation.component.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Component, OnInit, ViewChild, Input, Output, EventEmitter } from '@angular/core';
+import { Component, OnInit, ViewChild, Input, Output, EventEmitter, AfterViewInit } from '@angular/core';
```
Comment (costebano) [inline UI/src/app/components/sql-transformation/sql-transformation.component.ts:1]: remove unused import

**Comment by costebano** [comment]
please resolve conflicts

File: UI/src/app/components/sql-transformation/sql-transformation.component.ts (hunk 2 above)
Comment (MariaDolotova) [reply UI/src/app/components/sql-transformation/sql-transformation.component.ts]: Done

File: UI/src/app/components/pages/mapping/mapping.component.html (hunk 1 above)
Comment (MariaDolotova) [reply UI/src/app/components/pages/mapping/mapping.component.html:137]: Done

**Comment by MariaDolotova** [comment]
> please resolve conflicts

**Comment by MariaDolotova** [comment]
> please resolve conflicts

**Comment by MariaDolotova** [comment]
Conflicts resolved

**Review by costebano — APPROVED** [review]

**Pull request merged**